src/Servers/IIS/IIS/test/testassets/Directory.Build.targets

```diff
@@ -0,0 +1,48 @@
+<Project>
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory)..\, Directory.Build.targets))\Directory.Build.targets" />
+
+  <PropertyGroup>
+    <!-- Work around until we get a new WebSdk -->
```

Comment:
@Tratcher any context about what this workaround comment is talking about?

Reply:
I wondered about this too❕

Reply:
LOL, no, but it happened back in 3.0 so it's probably safe to say we got a fresh WebSdk update since then 😁.
https://github.com/dougbu/aspnetcore/commit/a474a0567f42412bf0972382bf4d41761b810be4

Reply:
The current default (in Microsoft.NET.Sdk.Web.ProjectSystem.targets) is `inprocess` and this override may make sense still. Let's use this space to answer

1. Should we update InProcessNewShimWebSite.csproj and InProcessWebSite.csproj instead of leaving this setting here❔
2. Did I break anything by effectively removing this setting from NativeIISSample.csproj and ServerComparison.TestSites.csproj❔
3. Does the setting have any impact in projects that import testsite.props but don't use `Microsoft.NET.Sdk.Web`❔
4. What are the best hosting models for each of the affected projects❔

Reply:
On (1), I suspect (given the names), the current setting is unhelpful for both InProcess*WebSite projects. Suggest we simply delete these lines and focus on the other projects.

On (2), "break" is probably too strong a word but I _did_ change things. Good thing tests are still working.